packit/api.py

```diff
@@ -426,20 +429,30 @@ def status(self):
         else:
             logger.info("\nGitHub upstream releases: No releases found.")
 
+    async def status_get_builds(self, status):
         builds = status.get_builds()
         if builds:
             logger.info("\nLatest builds:")
             for build in builds:
-                koji_builds_str = "\n".join(f" - {b}" for b in builds[build])
-                logger.info(f"{build}:\n{koji_builds_str}")
+                logger.info(f"{build}:\n{builds[build]}")
```

Comment:
```suggestion
                logger.info(f"{build}: {builds[build]}")
```
since there will always be just one

Reply:
+1, see also https://github.com/packit-service/packit/pull/306#discussion_r281724386